src/stress/StressEssentials.ps1

```diff
@@ -1,61 +1,227 @@
-﻿$workingDirectory=$env:TEMP
-$productArchitecture="TBD - passed in by VSTS likely."
-$productConfiguration="chk/ret: TBD - passed in by VSTS likely."
+# Ideally, we should use the current logged in user but this appears to require domain joining. 
 
-$productDirectory="$workingDirectory/product/$productArchitecture/$productConfiguration"
-$testsDirectory="$workingDirectory/tests/"
+param([String]$DropPat, [String]$WorkingDirectory, [Switch]$DebuggingBinaries = $false)
```

Comment:
Added a switch to the stress essentials. When invoking the stress essentials, if you pass in /DebuggingBinaries it will retrieve chk build of CoreCLR and debug builds of CoreFX.